src/Weather/Skysight/Skysight.cpp

```diff
@@ -427,6 +428,29 @@ Skysight::FromUnixTime(uint64_t t)
   return api.FromUnixTime(t);
 }
 
+void
+Skysight::Render(bool force_update)
+{
+  if (displayed_metric.metric) {
+    LogFormat("In Render!");
```

Comment:
Useless log message

Reply:
Agree. Will remove. I guess this is for debugging which I didn't do.